Modix/ClientApp/src/components/UserLookup/UserProfile.vue

```diff
@@ -38,6 +38,11 @@
                         <UserProfileField fieldName="Joined" :fieldValue="formatDate(user.joinedAt)" default="Never" />
                         <UserProfileField fieldName="Roles" :fieldValue="roles" allowHtml="true" />
                     </div>
+
+                    <div class="section-title">Messages by Channel</div>
+                    <div class="box" :v-if="messages">
+                        <PieChart :stats="messages" />
+                    </div>
```

Comment:
While testing this, I was getting a null ref exception occasionally that went away after I moved the `v-if` to a `div` around this section. So I'm thinking we'll probably want to make that change:

```html
<div v-if="messages">
    <div class="section-title">Messages by Channel</div>
    <div class="box">
        <PieChart :stats="messages" />
    </div>
</div>
```

Reply:
it didnt fix those errors for me, so I just added ` || []`